package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "afterlight-caves",
-  "version": "0.7.1",
+  "version": "0.8.1",
```

Comment:
Bumping the minor version resets the patch number, so this should be 0.8.0

Reply:
oops